google/gmail/tool.gpt

```diff
@@ -125,6 +125,26 @@ Param: draft_id: The ID of the draft email to send
 
 #!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/sendDraft.py
 
+---
+Name: List Attachments
+Description: List attachments in an email from a user's Gmail account.
+Credential: ../credential
+Share Context: Email List Context
+Param: email_id: The ID of the email to list attachments from
+
+#!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/listAttachments.py
+
+---
+Name: Get Attachment
+Description: Read an attachment from an email in a user's Gmail account.
+Credential: ../credential
+Share Context: Email List Context
+Share Tools: List Attachments
+Param: email_id: The ID of the email containing the attachment
+Param: attachment_id: The ID of the attachment to read
+
+#!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/getAttachment.py
```

Comment:
doesn't look like these files are included in the PR

Reply:
Oh, I forgot to push some files...pushed.